comment_iowriter.go

```diff
@@ -56,3 +59,86 @@ func (mc *UnifiedCommentWriter) Post(_ context.Context, c *Comment) error {
 	_, err := fmt.Fprintln(mc.w, s)
 	return err
 }
+
+var _ CommentService = &RDJSONLCommentWriter{}
+
+// RDJSONLCommentWriter
+type RDJSONLCommentWriter struct {
+	w io.Writer
+}
+
+func NewRDJSONLCommentWriter(w io.Writer) *RDJSONLCommentWriter {
+	return &RDJSONLCommentWriter{w: w}
+}
+
+func (cw *RDJSONLCommentWriter) Post(_ context.Context, c *Comment) error {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method RDJSONLCommentWriter.Post should have comment or be unexported
<!-- __reviewdog__:ChA3NmFhNGVjYjZkYjRlYjAwEg9nb2xpbnQtY2lyY2xlY2k= -->